app/models/ability.rb

```diff
@@ -76,9 +76,11 @@ def signed_in(user)
     can :manage, Event do |event|
       event.users.include?(user)
     end
-    # can create an event until the last day of a conference
-    can :create, Event, program_id: Conference.where('end_date >= ?', Date.today).map { |conference| conference.program.id}.compact
 
+    # can create an event if program has call for papers defined
+    can [:new, :create], Event do |event|
+      event.program.cfp_open? && event.new_record?
```

Comment:
@hennevogel, so my current implementation is okay? Since refining the rule overwrites the previous one and not supplements it, I should have a check for `new_record?`. Also, `:new` and `:create` action are independent of user ownership.